lms/templates/student_account/account_settings.underscore

```diff
@@ -1,26 +1,17 @@
 <main id="main" aria-label="Content" tabindex="-1">
 <div class="account-settings-container">
     <div class="wrapper-header">
-        <h2 class="header-title"><%- gettext("Account Settings") %></h2>
-        <small class="account-settings-header-subtitle"><%- gettext("These settings include basic information about your account. You can also specify additional information and see your linked social accounts on this page.") %></small>
+        <h3 class="header-title"><%- gettext("Account Settings") %></h3>
```

Comment:
There is no Heading level 2 that precedes this h3, so the logical structure of the page is broken.  This should be an H2.